packages/components/src/components/form/Select/index.tsx

```diff
@@ -388,20 +365,66 @@ const Select = ({
                         If we don't require the focused option to be on top, just delete 'await scrollToOption(lastOption);'
                         */
 
-                        // 1. scroll to the last option first and wait
-                        await scrollToOption(lastOption);
+                            // 1. scroll to the last option first and wait
+                            await scrollToOption(lastOption);
 
-                        // 2. scroll to the selected option
-                        scrollToOption(optionToFocusOn);
+                            // 2. scroll to the selected option
+                            scrollToOption(optionToFocusOn);
+                        }
                     }
                 }
             }
-        }
-    };
+        },
+        [findOption, scrollToOption, useKeyPressScroll],
+    );
+
+    // customize control to pass data-test attribute
+    const Control = useCallback(
+        (controlProps: ControlProps<Option, boolean>) => (
+            <components.Control
```

Comment:
can we rename `components` to `reactSelectComponents`? it took me quite a lot of time to find where it's defined, renaming will make it obvious

Reply:
done